lib/bugsnex/plug.ex

```diff
@@ -75,4 +77,21 @@ defmodule Bugsnex.Plug do
     {:ok, hostname} = :inet.gethostname
     to_string(hostname)
   end
+
+  defp filter_parameters(params) do
+    do_filter(params, Application.get_env(:bugsnex, :filter_params, @default_filter_params))
+  end
+
+  defp do_filter(%{__struct__: mod} = struct, _params) when is_atom(mod), do: struct
+  defp do_filter(%{} = map, params) do
+    Enum.into map, %{}, fn {k, v} ->
```

Comment:
Small thing you don't have to do, but I really like being more verbose aka `key`/`value` here :) Helps with readability a bit further down imo